admin/main.go

```diff
@@ -185,6 +185,59 @@ func verifyIDToken(app *firebase.App, idToken string) *auth.Token {
 	return token
 }
 
+// ==================================================================
+// https://firebase.google.com/docs/auth/admin/manage-sessions
+// ==================================================================
+
+func revokeRefreshTokens(app *firebase.App, uid string) {
+
+	// [START revoke_tokens]
+	ctx := context.Background()
+	client, err := app.Auth(ctx)
+	if err != nil {
+		log.Fatalf("error getting Auth client: %v\n", err)
+	}
+	if err := client.RevokeRefreshTokens(ctx, uid); err != nil {
+		log.Fatalf("error revoking tokens for user: %v, %v\n", uid, err)
+	}
+	// N.B the database API is not yet implemented in go, check the state of
```

Comment:
I think we should get rid of this DB update bit in each of the snippets, and just focus on the revocation part.